docker/test_dockerimage.sh

```diff
@@ -13,5 +13,15 @@ emcc --version
 java -version
 cmake --version
 
-# cleanup after test
-find ${EMSDK} -name "*.pyc" -exec rm {} \;
+exit_code=0
+
+# test emcc compilation
+echo 'int main() { return 0; }' | emcc -o /tmp/main.js -xc -
+node /tmp/main.js || exit_code=$?
+if [ $exit_code -ne 0 ]; then
+  echo "Node exited with non-zero exit code: $exit_code"
+  exit $exit_code
+fi
+
+# test embuilder
+embuilder build zlib --force
```

Comment:
Sorry I under standstand now, the cleanup was being done not inside the docker image but here in the `docker` subdirectory.

In that I case I think the old way was better.   No need to use `/tmp` just write the files here.

Reply:
Since I cannot mount volumes using the CircleCI Docker executor, writing the compiled output to the `docker` subdirectory is not possible. Would you rather use the CircleCI Docker executor (current state) or write to the compiled output to the `docker` subdirectory (and therefore using the `machine` executor)?

Reply:
Sorry, I think its fine to use the /tmp directory inside the container assuming that is what is happening?    Since that directory will be destroyed once the container finished running right?

Reply:
You're right. The `/tmp` is only used inside the container and then cleaned up afterwards once the container is finished (thanks to the `--rm` parameter).